src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -615,7 +596,7 @@ std::shared_ptr<GLHEVertArray> GetVertArray(EnergyPlusData &state, std::string c
 std::shared_ptr<GLHEResponseFactors> GetResponseFactor(EnergyPlusData &state, std::string const &objectName)
 {
     // Check if this instance of this model has already been retrieved
-    for (auto &thisRF : state.dataGroundHeatExchanger->responseFactorsVector) {
+    for (auto const &thisRF : state.dataGroundHeatExchanger->responseFactorsVector) {
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:618]:(style),[constVariable],Variable 'thisRF' can be declared as reference to const